db/migrate/20230713113935_create_users.rb

```diff
@@ -6,12 +6,14 @@ def change
       t.string :name
       t.text :photo
       t.text :bio
-      t.integer :update_user_posts_counter, default: 0
+      # t.integer :update_user_posts_counter, default: 0
       t.timestamps
     end
 
     add_index :users, :id
 
+    add_column :users, :posts_count, :integer, default: 0
```

Comment:
* This new change you implemented is causing an error in your integration tests as your tests try to create users using `posts_counter` but keeps clashing with this column being added `:posts_count` i strongly suggest that you ensure that the column name matches the one written in your schema and tests( which is `posts_counter`) 
![image](https://github.com/joyapisi/blog_app/assets/52497246/9391e3e5-7a2a-4a2c-bc42-e8c73324d631)
![image](https://github.com/joyapisi/blog_app/assets/52497246/0329839e-df0a-4d9b-a586-092bbffe0ff4)
![image](https://github.com/joyapisi/blog_app/assets/52497246/ad988e8b-f020-4112-8779-fc3fd08c8dfc)